docs/one-click-integrations/github-integration.md

```diff
@@ -100,10 +100,15 @@ You can also choose not to detect deployment automatically via GitHub and send y
 
 ## Automatic incident detection strategies {: id="incident-detection-strategy"}
 
-The Pulse GitHub integration detects incidents automatically using [pull request reverts](#gh-incident-pr-revert). You can also choose [not to detect incidents via GitHub](#gh-incident-not-detect).
+The Pulse GitHub integration can detect **incidents** automatically using the following strategies:
+
+-   [Pull request reverts](#gh-incident-pr-revert)
+-   [Pull request naming convention](#gh-incident-naming)
+
+You can also choose [not to detect incidents via GitHub](#gh-incident-not-detect).
 
 !!! note
-    Pulse incident detection mechanism bases on pull request. Thus, Pulse can only detect incidents automatically if you configured the deployment detection strategy to use [merged pull requests](#gh-deploy-merged-pr).
+    Pulse incident detection mechanism bases on pull requests. Thus, Pulse can only detect incidents automatically from GitHub if you configured the deployment detection strategy to use [merged pull requests](#gh-deploy-merged-pr).
```

Comment:
The initial sentence is a bit unclear. Here's a suggested edit (or we could also consider saying that `Pulse analyzes pull requests`):

```suggestion
    Pulse uses pull requests to detect incidents. Thus, Pulse can only detect incidents automatically from GitHub if you configured the deployment detection strategy to use [merged pull requests](#gh-deploy-merged-pr).
```

Reply:
Yep, much clear. Thanks!